hpi-archetype/pom.xml

```diff
@@ -40,18 +58,6 @@
     <url>http://github.com/jenkinsci/${project.artifactId}-plugin</url>
   </scm>
    -->
-  <repositories>
```

Comment:
Unless we are going to start publishing the new parent to central you will need to retain this to let people build a clean checkout without adding the repos to their settings.xml (never mind other side issues)